app/navigation/root-navigator.tsx

```diff
@@ -487,7 +487,7 @@ export const ContactNavigator: NavigatorType = () => {
   return (
     <StackContacts.Navigator>
       <StackContacts.Screen
-        name="Contacts"
+        name="contactList"
```

Comment:
There's also a reference to `"Contacts"` on line `564`.  Should we change that as well?